alarm/ScheduleTableViewCell.swift

```diff
@@ -21,13 +21,12 @@ class ScheduleTableViewCell: UITableViewCell {
       //activeSwitch.on = alarmEntity.enabled
     }
   }
-
+  
   override func awakeFromNib() {
     super.awakeFromNib()
     // Initialization code
-    self.contentView.userInteractionEnabled = false
```

Comment:
Hrmm, I feel like this was vestigial from me playing with `userInteractionEnabled` and trying to get the switches working. Did you see them working after dropping this line?